services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    movie_session = Ticket.objects.filter(movie_session=movie_session_id)
+    return [{"row": movie.row, "seat": movie.seat} for movie in movie_session]
```

Comment:
The variable `movie` in the list comprehension refers to `Ticket` objects, which might be misleading. Consider renaming it to something more descriptive, like `ticket`, to improve code readability.